packages/xforms-engine/src/body/appearance/selectAppearanceParser.ts

```diff
@@ -0,0 +1,38 @@
+import { TokenListParser, type ParsedTokenList } from '../../lib/TokenListParser.ts';
+
+export const selectAppearanceParser = new TokenListParser(
+	[
+		// From XLSForm Docs:
+		'compact',
+		'horizontal',
+		'horizontal-compact',
+		'label',
+		'list-nolabel',
+		'minimal',
+
```

Comment:
I see following appearances from XLSForm template missing here:

- no-buttons
- list
- image-map

Reply:
Ah yes, `compact` is deprecated. It was split up into `horizontal-compact` and `no-buttons` which can be combined if desired.

Reply:
If there are any known/canonical appearances (or aliases to them) that we want to add right away, please feel free to add them in! My hope is that this approach makes that trivial. I’d be happy to add these when my power is back, but you’re welcome to add them at your convenience as well.